iocore/aio/AIO.cc

```diff
@@ -220,7 +226,8 @@ ink_aio_init(ts::ModuleVersion v, AIOBackend backend)
   case AIOBackend::AIO_BACKEND_AUTO: {
     // detect if io_uring is available and can support the required features
     auto *ctx = IOUringContext::local_context();
-    if (ctx && ctx->supports_op(IORING_OP_WRITE) && ctx->supports_op(IORING_OP_READ)) {
+    if (ctx && ctx->valid()) {
+      setup_prep_ops(ctx);
```

Comment:
Now use_io_uring can be true even if the setup failed to initialize prep_ops?

Reply:
It should default to the always available readv/writev but I notice I use readv/writev for the READ supported case as well.  I'll fix that and make a comment to make it more clear.